src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/RecallDocumentServiceTest.kt

```diff
@@ -33,24 +32,6 @@ internal class RecallDocumentServiceTest {
     nomsNumber = NomsNumber("A1235B")
   )
 
-  @Test
```

Comment:
QQ @mikedunfordEE : reasoning for deleting this test please?

Reply:
see below